src/Microsoft.TestPlatform.CoreUtilities/Microsoft.TestPlatform.CoreUtilities.csproj

```diff
@@ -17,8 +17,7 @@
   </ItemGroup>
 
   <ItemGroup Condition=" '$(TargetFramework)' != '$(NetFrameworkMinimum)' ">
-    <PackageReference Include="System.Diagnostics.FileVersionInfo" Version="4.3.0" />
```

Comment:
This package is very old.  Dropping it in order to pick up System.Diagnostics.FileVersionInfo.dll via the implicit netcore app ref.